compiler/damlc/tests/daml-test-files/ProposalIou.EXPECTED.md

```diff
@@ -23,3 +23,4 @@
 <a name="type-proposaliou-proposaliou-81988"></a>**template instance** [ProposalIou](#type-proposaliou-proposaliou-81988)
 
 > = Proposal [Iou](#type-proposaliou-iou-51326)
+> 
```

Comment:
Not sure exactly where this whitespace comes from (I tried `stripEnd` from the doc after removing the template instance marker but no change). I can fix this later if it's an issue.

Reply:
I'm ok to merge it like this and fix it in a follow-up PR. Maybe @associahedron has an idea?